rosmon_core/src/launch/launch_config.cpp

```diff
@@ -423,6 +424,11 @@ void LaunchConfig::parseNode(TiXmlElement* element, ParseContext ctx)
 
 		node->setRespawnDelay(ros::WallDuration(seconds));
 	}
+        
+	if(shutdownHandler)
+	{
+		node->setShutdownHandler(ctx.evaluate(shutdownHandler));
```

Comment:
is this adding a shutdown handler for all cases? If rosnodes have their own shutdown handler are they called properly?

Reply:
This just executes whatever bash command is set, afterwards the node is killed how it is normally killed with rosmon, using kill()

Reply:
I can change the naming if its misleading

Reply:
no its fine, :+1: